Makefile

```diff
@@ -0,0 +1,12 @@
+include .env
+include secrets/.env
```

Comment:
is there a reason to put 2 env files ?

Reply:
this was to separate encrypted .env on ansible, but this is not necessary, you're right, I'll merge to keep it simple.